apps/python/descriptions/scart.rst

```diff
@@ -69,17 +87,32 @@ Examples
       scart -I [file.mseed] [SDS archive]
       scart -I [file.mseed] --with-filecheck [SDS archive]
 
-#. Collect data from an FDSNWS server using the :ref:`global_recordstream`
-   interface and write to a miniSEED file. The data streams and the time spans are
-   defined in a list file using the option ``list``. The list can be generated e.g.
-   by :ref:`scevtstreams`.
+#. Collect data using the :ref:`global_recordstream` interface e.g. (FDSNWS server)
+   and write to a miniSEED file or import it into a local :term:`SDS` archive. The
+   data streams and the time spans can be defined in several ways. The list of option
+   ``list`` can be generated e.g. by :ref:`scevtstreams`.
 
    .. code-block:: sh
 
+      scart -I fdsnws://[server]:80 --list list.file [SDS archive]
       scart -I fdsnws://[server]:80 --list list.file --stdout > file.mseed
+      scart -I fdsnws://[server]:80 -t '[start-time]~[end-time]' --nslc list.file [SDS archive]
+      scart -I fdsnws://[server]:80 -t '[start-time]~[end-time]' -n 'NET1,NET2' [SDS archive]
+      scart -I fdsnws://[server]:80 -t '[start-time]~[end-time]' -n 'NET' -c 'EH?' [SDS archive]
+      scart -I fdsnws://[server]:80 -t '[start-time]~[end-time]' -n 'N1.S1.L1.C1,N2.S2.L2.C2' [SDS archive]
+
+   It is possible to achieve the same result of the dump mode using a 
+   combination of the input mode and the :ref:`scmssort` command, which allows
+   to read the input data from any supported :ref:`global_recordstream`,
+   not only an SDS archive:
+
+   .. code-block:: sh
+
+      scart -I recordStream --list list.file --stdout | \
+        scmssort -u -E -v > file.mseed
 
 #. Check all files of an SDS archive or other directory structure for
-   miniSEED files with out-of-order records:
+   miniSEED files for out-of-order records:
```

Comment:
I believe "with" is more correct here.

Reply:
I should refrain from trying to correct english ;)